Cython/Compiler/Pythran.py

```diff
@@ -0,0 +1,154 @@
+from .PyrexTypes import BufferType, CType, CTypedefType, CStructOrUnionType
+
+_pythran_var_prefix = "__pythran__"
+# Pythran/Numpy specific operations
+def has_np_pythran(env):
+    scope = env.outer_scope
+    while scope != None:
+        scope = scope.outer_scope
+    ctx = env.global_scope().context
+    if ctx is None:
+        return False
+    options = ctx.options
+    return options is not None and options.np_pythran
+
+def is_pythran_supported_dtype(type_):
+    if isinstance(type_, CTypedefType):
+        return is_pythran_supported_type(type_.typedef_base_type)
+    return hasattr(type_,"is_numeric") and type_.is_numeric
```

Comment:
Fixed in https://github.com/cython/cython/pull/1607/commits/e3fb29c91b13edcc9297b02b0ca10e9a3da3928d